llvm/test/Transforms/InstCombine/minmax-fp-shared-op.ll

```diff
@@ -0,0 +1,16 @@
+; RUN: opt -passes=instcombine -S %s -o - | FileCheck %s
```

Comment:
Please use llvm/utils/update_test_checks.py to generate check lines.

Reply:
Manually writing check lines suffers from the same issue. We use UTC to avoid manually changing check lines the next time we implement a new fold/change the signature of intrinsics. I'd suggest adding a comment here to indicate this test is associated with a crash bug.